static/js/subs.js

```diff
@@ -923,10 +923,16 @@ exports.change_stream_name = function (e) {
             new_name_box.val('');
             ui.report_success(i18n.t("The stream has been renamed!"), $("#subscriptions-status "),
                               'subscriptions-status');
+            $("#name_change_error_" + stream_id).hide();
         },
         error: function (xhr) {
             ui.report_error(i18n.t("Error renaming stream"), xhr,
                             $("#subscriptions-status"), 'subscriptions-status');
+            new_name_box.attr("contenteditable", "true").focus();
+            $(e.target).parent().find(".checkmark").show();
+            $(e.target).parent().find(".editable").text("×");
```

Comment:
these selectors could be written more readably, e.g. using `sub_settings.find(".stream_name")` rather than `$(e.target).parent()`.